Missing buildLocalizeArrayFn module

I was trying to dynamically import locales from `date-fns` package and bumped into the following error: 

`Module not found: Can't resolve '../../../_lib/buildLocalizeArrayFn/index.js' in '/Users/ericreis/dev/vtex/test/node_modules/date-fns/locale/ar-DZ/_lib/localize'`

Here is the code I'm using to perform the dynamic import:

```js
const locale = 'en-GB'
import(`date-fns/locale/${locale}/index.js`).then(
  data => {
    console.log(data)
  })
```

I searched everywhere for this `buildLocalizeArrayFn`folder, but I didn't find it anywhere. Is it a bug? If not, could anyone point to me what I am doing wrong?

--- a/package.json
+++ b/package.json
@@ -81,6 +81,7 @@
     "node-fetch": "^1.3.3",
     "power-assert": "^1.3.1",
     "prettier": "^1.14.3",
+    "rimraf": "^2.6.3",
     "sinon": "^7.3.2",
     "size-limit": "^0.21.0",
     "snazzy": "^7.0.0",

--- a/scripts/build/package.sh
+++ b/scripts/build/package.sh
@@ -47,3 +47,4 @@ find "$dir" -type f -name "benchmark.js" -delete
 find "$dir/esm" -type f -name "package.json" -delete
 
 ./scripts/build/packages.js
+./scripts/build/removeOutdatedLocales.js $dir

--- a/scripts/build/removeOutdatedLocales.js
+++ b/scripts/build/removeOutdatedLocales.js
@@ -1,0 +1,20 @@
+#!/usr/bin/env node
+
+/**
+ * @file
+ * The script removes outdated locales from the package.
+ *
+ * It's a part of the build process.
+ */
+
+const path = require('path')
+const rimraf = require('rimraf')
+
+const packageDir = process.argv[2]
+if (!packageDir) throw new Error('Package dir should be passed as an argument')
+
+const locales = require('../../outdatedLocales.json')
+locales.forEach(locale => {
+  rimraf.sync(path.resolve(packageDir, `locale/${locale}`))
+  rimraf.sync(path.resolve(packageDir, `locale/esm/${locale}`))
+})

